js/src/APIClient.js

```diff
@@ -165,6 +165,26 @@ export class APIClient {
       }
     );
   }
+  async prepareSignature(contractId:string, fullName:string, accessToken:?string):Promise<object> {
+      return this.getCall('/signForMetamask/contract/' + contractId,  {
```

Comment:
should it be `/prepareSignature/contract/` here? (compared against routes in your related pr in openlaw backend https://github.com/openlawteam/openlaw/pull/1689/files)

Reply:
you're totally right. thanks!